en/README.md

```diff
@@ -2,7 +2,7 @@
 
 # Dronecode SDK
 
-[![Slack](https://px4-slack.herokuapp.com/badge.svg)](http://slack.px4.io)&nbsp;[![Discuss](https://img.shields.io/badge/discuss-DroneCore-ff69b4.svg)](http://discuss.px4.io/c/dronecore) [![jenkins build status](http://ci.px4.io:8080/buildStatus/icon?job=Dronecode/DronecodeSDK/develop)](http://ci.px4.io:8080/blue/organizations/jenkins/Dronecode%2FDronecodeSDK/branches/)
+[![Slack](https://px4-slack.herokuapp.com/badge.svg)](http://slack.px4.io)&nbsp;[![Discuss](https://img.shields.io/badge/discuss-Dronecode-ff69b4.svg)](https://discuss.px4.io/c/dronecode) [![jenkins build status](http://ci.px4.io:8080/buildStatus/icon?job=Dronecode/DronecodeSDK/develop)](http://ci.px4.io:8080/blue/organizations/jenkins/Dronecode%2FDronecodeSDK/branches/)

```

Comment:
This link seems wrong: https://discuss.px4.io/c/dronecode

I agree dronecore is not right, but isn't dronecode for discussions about dronecode - not the SDK?

Reply:
Good  point.

Reply:
And the subforum is actually called "Dronecode SDK" and the URL is `/sdk`, I just got that all wrong!